Specs/NumberWithUnit/Dutch/CurrencyModel.json

```diff
@@ -198,7 +198,15 @@
         "Text": "kwartje",
         "TypeName": "currency",
         "Resolution": {
-          "unit": "Euro",
+          "unit": "Netherlands guilder",
```

Comment:
Is this right? Doesn't it apply to Euro too?

Reply:
The Netherlands guilder had a 25 cent coin, nicknamed ‘kwartje’. With Euro we don’t use those nicknames originally. 

https://en.wikipedia.org/wiki/Twenty-five_cent_coin_(Netherlands)

Reply:
Then it looks good to me. I'll merge and we can refine if there's extra feedback. Thanks!